nameko/messaging.py

```diff
@@ -331,7 +328,10 @@ def connection(self):
             return   # don't cache a connection during introspection
 
         if self._connection is None:
-            self._connection = Connection(self.amqp_uri)
+            heartbeat = self.container.config.get(
```

Comment:
If this works with our oldest supported kombu then I think we're fine. It's ignored for non-pyamqp transports and has been supported there since 0.9.1.